src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,35 +4,100 @@
 
 @import '../includes/lib';
 
+$field-focus-ring: 0 0 0 2px rgba($color-blue-40, .5);
+$focus-border-color: #008aea;
+
+@mixin default-states {
+    &:focus {
+        border-color: $focus-border-color;
+    }
+
+    &:hover {
+        background-color: $color-marketing-gray-20;
+        border-color: $color-black;
+        box-shadow: none;
+        color: $color-black;
+    }
+
+    &:active {
+        background-color: $color-marketing-gray-20;
+        border-color: $color-marketing-gray-70;
+        color: $color-black;
+    }
+}
+
+@mixin default-states-dark {
+    &:focus {
+        border-color: $focus-border-color;
+    }
+
+    &:hover {
+        background-color: $color-dark-gray-60;
+        border-color: $color-white;
+        color: $color-white;
+    }
+
+    &:active {
+        background-color: $color-dark-gray-60;
+        border-color: $color-light-gray-50;
+        color: $color-white;
+    }
+}
+
+@mixin product-states {
+    &:focus {
+        border-color: $focus-border-color;
+    }
+
+    &:hover {
+        background-color: $color-blue-60;
+        border-color: transparent;
+        color: $color-white;
+    }
+
+    &:active {
+        background-color: $color-blue-80;
+        border-color: transparent;
+        color: $color-white;
+    }
+}
+
+@mixin product-states-dark {
+    @include product-states;
+}
+
 /* -------------------------------------------------------------------------- */
-// Primary Button
+// Button shape and size
 
 .mzp-c-button,
 a.mzp-c-button {
     @include border-box;
-    @include text-body-cta;
-    @include transition(background-color 100ms, box-shadow 100ms, color 100ms, transform 100ms);
```

Comment:
Should we keep the transition for the color changes?

Reply:
Can't hurt.